source/use-cases/sensitive-data-encryption.txt

```diff
@@ -13,26 +13,51 @@ Sensitive Data Encryption
+
+This document guides you through setup and implementation of Client-Side Field Level Encryption (CSFLE), a feature that encrypts document fields specified by a developer, in a example scenario to demonstrate how it can be used to secure your data.
+
+.. note::
+
+  Client-side Field Level Encryption is available starting in MongoDB 4.2 Enterprise only
 
 Problem
 ~~~~~~~
 
-Pellentesque non mi et est congue mattis pretium sit amet mi. Praesent
-nec tortor nec arcu molestie dapibus. Phasellus at nunc ut quam molestie
-finibus. Suspendisse finibus lectus eget nulla eleifend, vel consequat
-leo rhoncus. Phasellus lacinia tortor id aliquet condimentum. Morbi
-sodales est a leo rhoncus, non pretium neque dignissim.
+In this scenario, we secure sensitive data on a Medical Care Management System which stores patients' personal information, appointments, and vital sign measurement reports for a fictitious company, *MedcoMD*. While none of this data should ever be exposed to the public, if a malicious user gained access to data such as the patient's social security number (SSN) or their vital sign measurements, it would be disastrous and damaging to the company.
+
+MedcoMD provides access to this system for two types of users:
+
+1.  Receptionists, who use the system to schedule appointments for Patients, identified by a combination of contact information and their SSN.
+2.  Doctors, who use the system to access and enter Patients' vital sign measurements
+
+While MedcoMD trusts their employees never to retrieve sensitive data, there are several ways it could be exposed if left unencrypted including by leaving it exposed on a screen or print-out, by sniffing the data from network packets on an insecure network, or by obtaining unauthorized access to the database. If they store complete, encrypted documents on the server, every query would require decrypting the entire collection which is too computationally expensive, and therefore, slow.
+
+What can MedcoMD do to balance the functionality and access restrictions of their Medical Care Management System?
 
 Solution
 ~~~~~~~~
 
-Praesent sit amet elit pretium mi molestie porta a sit amet ipsum. Sed
-sollicitudin purus et mi blandit sodales.
+MongoDB CSFLE allows the developer to specify the sensitive fields of a document to keep encrypted on the server. The client encrypts the specified, sensitive fields upon writes and transparently decrypts them upon access when necessary. The collections can be queried by any of the unencrypted fields without additional cost, and encrypted fields can be queried -- with restrictions documented in the following section, :ref:`DeterministicVsRandom` -- with minimal overhead.
+
+If an attacker manages to gain access to the MongoDB database, unless they also gain access to an authorized client, they will only be able to see the sensitive fields in encrypted form.
```

Comment:
nit: Doesn't necessarily have to be an "attacker". e.g. it could just be the janitor seeing data on screen non-maliciously - not an "attack" but still very uncool.

Maybe "unauthorized user"?

Reply:
Makes sense, will update.